### ISSUE TITLE
Unblock "Illustria"

### ISSUE THREAD

**Issue by PikaMug** [Minehut/Meta#548]
### Filter request

**Checklist**
- [x] I have read and agree to the README (found on https://github.com/Minehut/Meta)
- [x] I have ensured the phrase is not properly filtered on the platform

**Platform**

Website

**Should the phrase be filtered or unfiltered?**

Unfiltered

**Phrase**

```
Illustria
```

**Comment by CoolProgrammerX** [comment]
Hey, sorry to reply to this post for no reason, but I have a suggestion for you. In the future, you can use the Minehut Meta discord server's #filter-request channel to ask for a filter change. That way, you won't have to spend time in making a new issue on GitHub, and instead just a small message on the discord server. Good luck!

**Comment by awaffle1** [comment]
> Hey, sorry to reply to this post for no reason, but I have a suggestion for you. In the future, you can use the Minehut Meta discord server's #filter-request channel to ask for a filter change. That way, you won't have to spend time in making a new issue on GitHub, and instead just a small message on the discord server. Good luck!

See #533

**Comment by PikaMug** [comment]
Yes, I mean the filter on server names. I've just submitted the request on the Meta Discord as well.

**Comment by PikaMug** [comment]
Ten days later, I still get "Please pick an appropriate name." I've no choice but to use a competitor. This matter is now closed.

**Comment by quick007YT** [comment]
lmao
